Task_DEV_5/Task_DEV_5/SecondSorter.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class SecondTextSort
+    {
+        private string path { get; set; }
+        public FileStream fstream;
```

Comment:
For fields more preferable private modificator. For properties - public modificator.

Reply:
Done.